Project3/style.css

```diff
@@ -0,0 +1,152 @@
+figure {
+    position: relative;
+
+
+}
+
+figure img {
+    transition: all 1s;
+    width: 200px;
+    height: auto;
+    margin: 0%;
+}
+
+figure img:hover {
+    transform: scale(1.3);
+}
+
+figcaption {
+    display: none;
+    text-align: left;
+    font-family: 'Heebo', sans-serif;
+    position: relative;
+    width: 50%;
+
+}
+
+h1 {
+    font-family: 'Heebo', sans-serif;
+    text-align: center;
+    color: rgb(255, 255, 255);
+    font-size: 100px;
+    font-display: fallback;
```

Comment:
This isn't how `font-display` is used, FYI:
```suggestion
```

This property gets used by web fonts [like so](https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-display).